careerfairsystems/arkad-app-2018
# #86 add splash screen for ios

## Change description

This adds a splash screen for most iPhones.

## How to verify

Start the application and notice that the splash screen is present on multiple iPhone models..

## Issues fixed

This fixes #86
